### PR TITLE
[CLEANING] remove unused imports

### PR DESCRIPTION
**Pull request by ppa-odoo**
Removed unused import related to product in modules of  crm, sale_crm and point_of_sale.

### CODE REVIEW

File: addons/point_of_sale/res_users.py [hunk 1]
```diff
@@ -3,8 +3,6 @@
```
Comment (rim-odoo) [inline addons/point_of_sale/res_users.py]: the import math seems to be removed

File: addons/point_of_sale/res_partner.py [hunk 2]
```diff
@@ -3,8 +3,6 @@
```
Comment (rim-odoo) [inline addons/point_of_sale/res_partner.py]: the import math seems to be removed

Comment (ppa-odoo) [inline addons/point_of_sale/res_partner.py]: @rim-odoo, i have removed.

File: addons/point_of_sale/res_users.py (hunk 1 above)
Comment (ppa-odoo) [inline addons/point_of_sale/res_users.py]: @rim-odoo, i have removed.

**Pull request merged**